ftw/theming/extensions.py

```diff
@@ -53,15 +53,21 @@ def find_relative_resource(relpath):
     return path
 
 
+def chunks(l, n):
+    if not l:
+        return []
+    return [l[i:i + n] for i in xrange(0, len(l), n)]
+
+
 def fill_svg(data, fill_css=None, fill_xpath=None):
     doc = etree.fromstring(data)
     expressions = []
 
-    for xpr, color in fill_xpath or ():
+    for xpr, color in chunks(fill_xpath, 2) or ():
```

Comment:
This code here should be backwards compatible with the old code too. I may not want to update the other packages in the project because there may be other changes I do not want.

If we cannot make it backwords compatible we will not merge it.